tutorials/real-time-analytics-tutorial.rst

```diff
@@ -0,0 +1,153 @@
+
+::
+
+    docker cp users.csv citus_master:.
+    docker cp events.csv citus_master:.
+            
+Creating tables 
+---------------
+                                                                                             
+To start, you can first connect to the Citus coordinator using psql.
+
+**If you are using Citus Cloud**, you can connect by specifying the connection string (URL in the formation details):
+    
+::
+    
+    psql connection-string
+
+Please note that certain shells may require you to quote the connection string when connecting to Citus Cloud. For example, :code:`psql "connection-string"`.
+
+**If you are using Docker**, you can connect by running psql with the docker exec command:
+
+::
+    
+    docker exec -it citus_master psql -U postgres
+
+Then, you can create the tables by using standard PostgreSQL :code:`CREATE TABLE` commands.
+
+::
+
+    CREATE TABLE github_events                                                                   
+    (                                                                                            
+        event_id bigint,                                                                         
+        event_type text,                                                                         
+        event_public boolean,                                                                    
+        repo_id bigint,                                                                          
+        payload jsonb,                                                                           
+        repo jsonb,                                                                              
+        user_id bigint,                                                                          
+        org jsonb,                                                                               
+        created_at timestamp                                                                     
+    );                                                                                           
+
+    CREATE TABLE github_users                                                                    
+    (                                                                                            
+        user_id bigint,                                                                          
+        url text,                                                                                
+        login text,                                                                              
+        avatar_url text,                                                                         
+        gravatar_id text,                                                                        
+        display_login text                                                                       
+    );
+
+Next, you can create indexes on events data just like you would do in PostgreSQL.
```

Comment:
Do we want to add a line about using GIN indexes? It could say something around postgres allowing you to index json columns and query them efficiently.